src/app/admin/adminActions.ts

```diff
@@ -4,15 +4,22 @@ import { createServerActionClient } from "@supabase/auth-helpers-nextjs";
 import { cookies } from "next/headers";
 import { FunctionsResponse } from "@supabase/functions-js/src/types";
 import { AdminUserAttributes } from "@supabase/gotrue-js";
+import { UserResponse } from "@supabase/supabase-js";
+import { Database } from "@/databaseTypesFile";
 
-const supabase = createServerActionClient({ cookies });
+const supabase = createServerActionClient(
```

Comment:
Note: After Mina's VFB-170 is merged, we'll change this to use getSupabaseAdminAuthClient.